index.js

```diff
@@ -0,0 +1,250 @@
+// MINIMUM HW TASK 
+
+// 1 task 
+// User age
+let userAge = prompt("Please enter your age:");
+
+// input is valid
+if (isNaN(userAge) || userAge === null || userAge.trim() === "" || userAge < 0) {
+    alert("Please enter a valid age.");
+} else {
+    userAge = Number(userAge);
+    
+    // Determine the user's category based on their age
+    if (userAge >= 0 && userAge <= 11) {
+        alert("You are a child.");
+    } else if (userAge >= 12 && userAge <= 17) {
+        alert("You are a teenager.");
+    } else if (userAge >= 18 && userAge <= 59) {
+        alert("You are an adult.");
+    } else if (userAge >= 60) {
+        alert("You are a senior.");
+    } else {
+        alert("Please enter a valid age.");
+    }
+}
+
+
+// 2 task 
+let singleDigitNumber = prompt("Please enter a number from 0 to 9:");
+
+// Object to map numbers to special characters
+const specialCharacters = {
+    0: ')',
+    1: '!',
+    2: '@',
+    3: '#',
+    4: '$',
+    5: '%',
+    6: '^',
+    7: '&',
+    8: '*',
+    9: '('
+};
```

Comment:
better to use Array in this case
```suggestion
const specialCharacters = [')', '!', '@', '#', '$', '%', '^', '&', '*', '('];
```